screenshot/src/index.js

```diff
@@ -35,13 +52,32 @@ function fitText() {
   })
 }
 
+function updateName(template, firstname, lastname) {
+  return template
+    .replace('##FIRSTNAME##', firstname)
+    .replace('##LASTNAME##', lastname)
+}
+
+function generatePNG() {}
+
 async function main() {
   const template = await fetchTemplate()
-  await appendTemplate(template)
+  await appendTemplate(updateName(template, 'Kevin', 'Simper'))
   fitText()
-  setTimeout(() => {
-    generateCanvas()
+  setTimeout(async () => {
```

Comment:
Settimeout is used because I manipulate the DOM and I didn't implement a promise 😅 can be fixed